src/opening-period/time-span/TimeSpan.scss

```diff
@@ -0,0 +1,45 @@
+@import 'node_modules/hds-design-tokens/lib/all.scss';
+
+.remove-time-span-button {
+  color: var(--color-coat-of-arms);
+}
+
+.start-and-end-time-container {
+  display: flex;
+  align-items: flex-end;
+}
+
+.dash-between-start-and-end-time-container {
+  height: 56px;
```

Comment:
This seem to be redundant declaration

Reply:
- [ ] Remove redundant height style declaration